xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/main/java/org/xwiki/rendering/macro/descriptor/DefaultContentDescriptor.java

```diff
@@ -65,9 +74,22 @@ public DefaultContentDescriptor(boolean mandatory)
      * @param mandatory indicate if the content is mandatory.
      */
     public DefaultContentDescriptor(String description, boolean mandatory)
+    {
+        this(description, mandatory, null);
```

Comment:
This should be consistent with the default implementation in `ContentDescriptor`. Either `null` (if we think it's better to make explicit the content type is unknown) for both or `String.class` (if we feel String.class to be more consistent with parameters types logic) for both.